### PR TITLE
Enable rbac

### PR DESCRIPTION
**Pull request by piotrmsc**
## Proposed changes
Enable RBAC in microk8s to properly test charts

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [ ] I have read the [security policy](../security/policy)
- [ ] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security
vulnerability, I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation within the code base (if appropriate)

### CODE REVIEW

**Comment by aeneasr** [comment]
LMK if you need help with this

**Comment by piotrmsc** [comment]
Do you have any additional ideas on how to solve file permissions issue?

If this will fail we will have to think about going back to KIND

**Comment by aeneasr** [comment]
We could probably try another reboot of 1.14 microk8s with the workarounds mentioned in the document you shared with me on slack?

**Comment by piotrmsc** [comment]
yeah, I may try to do it already on this PR but I will be able to get back to it later this day or tomorrow :)

**Comment by piotrmsc** [comment]
@aeneasr ready for review! I have enbaled rbac and bumped microk8s to 1.15 it has a few hacks, pls checkout my comments :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,15 +35,14 @@ jobs:
       - run:
           name: "Start microk8s"
           command: |
-            sudo microk8s.start
-            sudo microk8s.status --wait-ready
-            sudo microk8s.enable dns
-            kubectl get all --all-namespaces
+            (echo "microk8s.start; microk8s.status --wait-ready; microk8s.enable dns rbac ") | sudo -H -s -E
```
Comment (piotrmsc) [inline .circleci/config.yml]: I had to use echo to pipe commands once sudo -H -s -E is executed

File: .circleci/config.yml [hunk 2]
```diff
@@ -35,15 +35,14 @@ jobs:
...
-            sudo microk8s.enable dns
-            kubectl get all --all-namespaces
+            (echo "microk8s.start; microk8s.status --wait-ready; microk8s.enable dns rbac ") | sudo -H -s -E  
       - run: 
           name: "Install Helm"
-          command: |
+          command: |            
             curl -L https://git.io/get_helm.sh | bash
-            helm init --service-account default --wait
+            (echo "kubectl create serviceaccount --namespace kube-system tiller; kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller") | sudo -H -s -E
```
Comment (piotrmsc) [inline .circleci/config.yml]: same story here, otherwise no access to kubectl even with `sudo kubectl ...`

File: .circleci/config.yml (hunk 1 above)
Comment (aeneasr) [reply .circleci/config.yml]: nice!

File: .circleci/config.yml [hunk 3]
```diff
@@ -35,15 +35,14 @@ jobs:
...
           command: |
-            sudo microk8s.start
-            sudo microk8s.status --wait-ready
-            sudo microk8s.enable dns
-            kubectl get all --all-namespaces
+            (echo "microk8s.start; microk8s.status --wait-ready; microk8s.enable dns rbac ") | sudo -H -s -E  
       - run: 
           name: "Install Helm"
-          command: |
+          command: |
```
Comment (aeneasr) [inline .circleci/config.yml]: Not sure if these whitespaces can cause issues but it's yaml so maybe:

```suggestion
          command: |
```

**Pull request merged**